redpen-core/src/main/java/cc/redpen/validator/sentence/SpaceBetweenAlphabeticalWordValidator.java

```diff
@@ -40,7 +40,9 @@
     private final Pattern pat = Pattern.compile(shard + "\\s+(" + word + ")\\s+" + shard);
 
     public SpaceBetweenAlphabeticalWordValidator() {
-        super("forbidden", false); // Spaces are enforced (false) or forbidden (true)
+        super("forbidden", false, // Spaces are enforced (false) or forbidden (true)
+              "skip_before_chars", "",
```

Comment:
I would like to shorten the configuration names.  If you are ok, could you please rename ```skip_before_chars``` into ```skip_before``` and  ```skip_after_chars``` to ```skip_after```?